Tools/PowershellModule/doc/new-winget-rest-source-azure.md

```diff
@@ -556,15 +535,12 @@ For more information on Azure Functions, visit their Docs article: [Introduction
         - Value: [Value]
         - Deployment slot setting: Disabled
 
-#### Import Windows Package Manager API
-
-**How to:**
+### Import Windows Package Manager API
 
 1. Open a PowerShell window on your computer.
 
     ``` PowerShell
-    PS C:\> $ArchiveFunctionZip = "C:\Projects\winget-cli-restsource\src\WinGet.RestSource.Infrastructure\CompiledFunctions.zip"
+    PS C:\> $RestAPI = "C:\Projects\winget-cli-restsource\winget-cli-restsource.zip"
```

Comment:
what is this path?  Should this be a pointer to the release download for the azure functions?
If you expect people to be building the solution locally, this should be pointing to how to generate the packaged azure functions output locally?

Reply:
I'm confused.. this is the retrieved ZIP from releases... done locally?